homeassistant/components/aseko_pool_live/sensor.py

```diff
@@ -40,6 +40,13 @@ class AsekoSensorEntityDescription(SensorEntityDescription):
         state_class=SensorStateClass.MEASUREMENT,
         value_fn=lambda unit: unit.air_temperature,
     ),
+    AsekoSensorEntityDescription(
+        key="electrolyzer",
+        translation_key="electrolyzer",
+        native_unit_of_measurement="g/h",
```

Comment:
G/h? What does that mean?

Reply:
Hi, it is gas (chlorine) production in grams per hour (power of the electrolyzer) for the ASEKO Salt units:
![Screenshot_20241002_090345_Chrome](https://github.com/user-attachments/assets/c0e4c24c-1c28-49ad-82b1-c44d841353e3)

Reply:
Should make it more clear? `gram/hour`

Reply:
The native app & web app show it as g/h, so I wanted to be consistent with those.